llvm/test/CodeGen/AMDGPU/GlobalISel/llvm.amdgcn.sdot4.ll

```diff
@@ -41,19 +41,19 @@ define i32 @v_sdot4_cast_v4i8(<4 x i8> %a, <4 x i8> %b, i32 %c) {
 ; GFX906-LABEL: v_sdot4_cast_v4i8:
 ; GFX906:       ; %bb.0:
 ; GFX906-NEXT:    s_waitcnt vmcnt(0) expcnt(0) lgkmcnt(0)
-; GFX906-NEXT:    s_mov_b32 s5, 8
-; GFX906-NEXT:    s_movk_i32 s4, 0xff
-; GFX906-NEXT:    v_lshlrev_b32_sdwa v1, s5, v1 dst_sel:DWORD dst_unused:UNUSED_PAD src0_sel:DWORD src1_sel:BYTE_0
-; GFX906-NEXT:    v_and_or_b32 v0, v0, s4, v1
+; GFX906-NEXT:    v_mov_b32_e32 v10, 8
```

Comment:
Lots of GlobalISel tests show regressions like this where constants are materialized in vgprs instead of sgprs. This does not seem to affect SelectionDAG.

Reply:
Really we need a GlobalISel solution for picking the register banks of constants. I wouldn't worry about these for now